pages/webview/content.py

```diff
@@ -140,6 +144,46 @@ def click_get_this_book_button(self):
         self.get_this_book_button.click()
         return self.GetThisBook(self).wait_for_region_to_load()
 
+    @property
+    def books(self):
+        return self.Books(self)
+
+    class BookContents(Region):
+        _root_locator = (By.CSS_SELECTOR, '.booksContaining')
+        _overview_locator = (By.CLASS_NAME, "title")
+        _book_locator = (By.XPATH, './ul/li')
+
+        @property
+        def is_displayed(self):
+            return self.is_element_displayed(*self._book_locator)
+
+        @property
+        def description(self):
+            self.find_element(*self._root_locator)
+            return self.find_element(*self._overview_locator).text
+
+        @property
+        def books(self):
+            elements = self.find_elements(*self._book_locator)
+            return [self.Book(self.page, element) for element in elements]
+
+        class Book(Region):
+            _author_locator = (By.XPATH, ".//ul/li[1]//b")
+            _revision_date_locator = (By.XPATH, ".//ul/li[2]//b")
+            _go_to_book_locator = (By.XPATH, ".//ul/li[3]//a")
```

Comment:
Please avoid using XPATH. you should be able to traverse down to the child element using the CSS Selector.